abdiff/core/run_ab_transforms.py

```diff
@@ -142,6 +175,22 @@ def wait_for_containers(containers: list[Container], timeout: int = 180) -> None
         time.sleep(0.25)
 
 
+def aggregate_logs(run_directory: str, containers: list[Container]) -> str:
+    """Retrieve logs for containers in a list, aggregating to a single log file."""
+    log_file = str(Path(run_directory) / "transformed/logs.txt")
```

Comment:
Looks like this `Path` call was missed when you removed the others

Reply:
(Extending / referencing the discussion from slack)

I think `Path` here _does_ make sense, as we are **constructing** a new Path from components.